packages/fxa-settings/src/pages/Signin/SigninRecoveryMethod/en.ftl

```diff
@@ -0,0 +1,11 @@
+## Signin recovery method page
+## This page is shown to users when they are having trouble signing in with
+## their password, and they previously had set up an account recovery method.
+
+signin-recovery-method-header = Sign in
+signin-recovery-method-subheader = Choose a recovery method
+signin-recovery-method-details = Let’s make sure it’s you using your recovery methods.
+signin-recovery-method-phone = Recovery phone
+signin-recovery-method-code = Authentication codes
+# Variable: $numberOfCodes (String) - The number of authentication codes the user has left, e.g. 4
+signin-recovery-method-phone-info = { $numberOfCodes } of 5 available
```

Comment:
I don't see this string referenced anywhere. Is it used?

Reply:
This _should_ have been `signin-recovery-method-code-info` so another good catch. I'll be updating anyway though: https://github.com/mozilla/fxa/pull/18180#discussion_r1904724056